website/content/api-docs/system/internal-ui-authenticated-messages.mdx

```diff
@@ -10,6 +10,9 @@ description: >-
 The `/sys/internal/ui/authenticated-messages` endpoint is used by the UI to
 retrieve the active post-login custom messages so that it can display them.
 
+You must install Vault Enterprise 1.16.0 or higher to create custom messages
```

Comment:
This reads a little strange to me, I think it might make more sense to have something like:

```
To create custom messages that can be retrieved using this endpoint,
use [the custom messages API](link/to/the/API).
```
And for the 'this is Enterprise specific' information to be on the Enterprise specific API page.

All that being said, this is stylistic, so feel free to ignore it, I just wanted to offer an alternative.

Reply:
That formulation is used elsewhere and is what Finn asked me to use during our Sprint Planning meeting.

Reply:
Works for me! Thanks for clarifying :)